client/src/components/RequestForm.js

```diff
@@ -1,83 +1,139 @@
 import styled from "@emotion/styled";
 import React from "react";
-import colors from "../utils/colors";
+import PropTypes from "prop-types";
+import { Dropdown } from "./Dropdown";
+import { Input } from "./Input";
+import { FloatingLabel } from "./FloatingLabel";
+import { ButtonFull } from "./ButtonFull";
+import { Textarea } from "./Textarea";
+
+export default function RequestForm() {
+  const [artist, setArtist] = React.useState("");
+  const [date, setDate] = React.useState("");
+  const [location, setLocation] = React.useState("");
+  const [city, setCity] = React.useState("");
+  const [arrival, setArrival] = React.useState("");
+  const [price, setPrice] = React.useState("");
+  const [text, setText] = React.useState("");
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const request = {
+      artist,
+      date,
+      location,
+      city,
+      arrival,
+      price,
+      text,
+    };
+
+    alert(JSON.stringify(request));
+  }
 
-export default function ConcertCardBig() {
   return (
-    <Form type="submit">
+    <Form onSubmit={handleSubmit}>
       <Div>
-        <Label>Band/Künstler</Label>
-        <Input type="text" placeholder=""></Input>
+        <Input
+          required
+          value={artist}
+          onChange={(event) => {
+            setArtist(event.target.value);
+          }}
+        />
+        <FloatingLabel>Band/Künstler</FloatingLabel>
       </Div>
       <Div>
-        <Label>Datum</Label>
-        <Input type="date" placeholder=""></Input>
+        <Input
+          required
+          type="date"
+          value={date}
+          onChange={(event) => {
+            setDate(event.target.value);
+          }}
+        />
+        <FloatingLabel>Datum</FloatingLabel>
       </Div>
       <Div>
-        <Label>Location</Label>
```

Comment:
everywhere else you use german but here it's english. maybe try to stick to one language...